ci-operator/step-registry/baremetal/lab/pre/load-balancer/baremetal-lab-pre-load-balancer-commands.sh

```diff
@@ -23,9 +24,12 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   fi
 
   if [[ "$name" =~ bootstrap* ]] || [[ "$name" =~ master* ]]; then
-    MC="$MC
+    MC22623="$MC22623
       server $name $ip:22623 check inter 1s
       server $name-v6 [$ipv6]:22623 check inter 1s"
+    MC22624="$MC22624
```

Comment:
If this change is committed, we need to make sure it is included in the documentation as the load balancer is normally a day0 operation (and the current doc doesn't expose this port). Current users will need to know about the need to update their confs before running this day2 op.

Moreover, we were discussing of whether exposing this port publicly is the right path and is secure. 
@bmanzari @andfasano what do you think?

Reply:
From a general point of view we should rely on the already existing documentation for adding  a new node, since we're not introducing any new/special behavior. From a strictly speaking testing point of view, my preference would be that the test environment will expose and use just the secure port (22623), since it's the one selected by default now by ABI when adding a new node